todo_app/app.py

```diff
@@ -3,30 +3,38 @@
 from todo_app.flask_config import Config
 from todo_app.data.trello_items import get_cards, post_add, update_status, get_card, delete_card, get_lists
 from todo_app.data.Item import Item
-app = Flask(__name__)
-app.config.from_object(Config())
+from todo_app.data.viewmodel import ViewModel
 
 
-@app.route('/')
-def index():
-    right_list = get_lists()[-1]["name"]
-    items=get_cards()
-    items=sorted (items,key=lambda item: item.status, reverse=True)
-    return render_template ('index.html', items=items, right_list=right_list)
 
-@app.route('/Add', methods=['POST'])
-def Add():
-    title = request.form.get ("title")
-    post_add (title)
-    return redirect (url_for("index"))
+def create_app():
+    app = Flask(__name__)
+    #app.config.from_object(Config())#
```

Comment:
Delete commented out code. And if you're going to delete this, you may as well delete flask_config.py and the import statement as well.

Note that we will want a SECRET_KEY again in module 10. So I don't mind if you want to leave it in for convenience.